distribution/tools/launchers/src/main/java/org/elasticsearch/tools/launchers/JvmOptionsParser.java

```diff
@@ -91,7 +92,10 @@ public void accept(final int lineNumber, final String line) {
                 substitutePlaceholders(jvmOptions, Map.of("ES_TMPDIR", System.getenv("ES_TMPDIR")));
             final List<String> ergonomicJvmOptions = JvmErgonomics.choose(substitutedJvmOptions);
             substitutedJvmOptions.addAll(ergonomicJvmOptions);
```

Comment:
Since you concat these two's `Stream`'s below, I don't think you need to add these here (or you could not `concat` them below)?